Tests/PubNubTests/Networking/Routers/ChannelGroupEndpointTests.swift

```diff
@@ -202,29 +200,29 @@ final class ChannelGroupsEndpointTests: XCTestCase {
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
 
-  // MARK: - Add Group Channel
+// MARK: - Add Channel to Group
 
-  func testGroupChannelsAdd_Endpoint() {
-    let endpoint = Endpoint.addChannelsForGroup(group: testGroupName, channels: testChannels)
+extension ChannelGroupsRouterTests {
+  func test_AddChannelsForGroup_Router() {
+    let router = ChannelGroupsRouter(.addChannelsToGroup(group: testGroupName, channels: testChannels), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 126 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)